content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,169 @@ owner: istio/wg-security-maintainers
+
+While this is required for incremental adoption or allowing traffic from clients without an Istio sidecar, it also weakens the security stance.
+It is recommended to [migrate to strict mode](/docs/tasks/security/authentication/mtls-migration/) when possible, to enforce that mutual TLS is used.
+
+Mutual TLS alone is not always enough to fully secure traffic, however, as it provides only authentication, not authorization.
+This means that anyone with a valid certificate can still access a service.
+
+To fully lock down traffic, it is recommended to configure [authorization policies](/docs/tasks/security/authorization/).
+These allow creating fine-grained policies to allow or deny traffic. For example, you can allow only requests from the `app` namespace to access the `hello-world` service.
+
+## Understand traffic capture limitations
+
+The Istio sidecar works by capturing both inbound traffic and outbound traffic and directing them through the sidecar proxy.
+
+However, not *all* traffic is captured:
+
+* Redirection only handles TCP based traffic. Any UDP or ICMP packets will not be captured or modified.
+* Inbound capture is disabled on many [ports used by the sidecar](/docs/ops/deployment/requirements/#ports-used-by-istio) as well as port 22. This list can be expanded by options like `traffic.sidecar.istio.io/excludeInboundPorts`.
+* Outbound capture may similarly be reduced through settings like `traffic.sidecar.istio.io/excludeOutboundPorts` or other means.
+
+In general, there is minimal security boundary between an application and its sidecar proxy. Configuration of the sidecar is allowed on a per-pod basis, and both run in the same network/process namespace.
+As such, the application may have the ability to remove redirection rules and remove, alter, terminate, or replace the sidecar proxy.
+This allows a pod to intentionally bypass its sidecar for outbound traffic or intentionally allow inbound traffic to bypass its sidecar.
+
+As a result, it is not secure to rely on all traffic being captured unconditionally by Istio.
+Instead, the security boundary is that a client may not bypass *another* pod's sidecar.
+
+For example, if I run the `reviews` application on port `9080`, I can assume that all traffic from the `productpage` application will be captured by the sidecar proxy,
+where Istio authentication and authorization policies may apply.
+
+### Defense in depth with `NetworkPolicy`
+
+To further secure traffic, Istio policies can be layered with Kubernetes [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+This enables a strong [defense in depth](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) strategy that can be used to further strengthen the security of your mesh.
+
+For example, you may choose to only allow traffic to port `9080` of our `reviews` application.
```

Comment:
It would be good to try to link to some examples we have in other docs, e.g., https://istio.io/latest/docs/tasks/traffic-management/egress/egress-gateway/#apply-kubernetes-network-policies

Reply:
Never mind, I see you added it below.